test/binCases/errors/info-verbosity/stdin.js

```diff
@@ -1,7 +1,6 @@
 "use strict";
 
 module.exports = function testAssertions(code, stdout, stderr) {
-	expect(stderr[3]).toContain("Error: Invalid configuration object.");
-	expect(stderr[4]).toContain("configuration['info-verbosity'] should be one of these:");
-	expect(stderr[5]).toContain("\"none\" | \"info\" | \"verbose\"");
+	expect(stderr[stderr.length - 3]).toContain("Invalid values:");
+	expect(stderr[stderr.length - 2]).toContain("  Argument: info-verbosity, Given: \"false\", Choices: \"none\", \"info\", \"verbose\"");
```

Comment:
Maybe this could be done without slicing at the end of the array? If you already know the size of it?

Reply:
Total length will vary because adding more options into `yargs.options` will change the total length as invalid options do print out the whole usage info, but i know that -3 and -2 will be error specific values that i want to check

Reply:
gotcha, but the tests are static, yes? So the size will always be constant

Reply:
Unless there's 9 different tests for 1 case here ?

Reply:
yeah sorry, so any update to their description or removal\addition will change lines amount

Reply:
right. Let's keep it for now, could change it later if needed

Reply:
true, thank you!